client/my-sites/email/inbox/mailbox-selection-list/style.scss

```
@@ -5,6 +5,68 @@ body.is-section-inbox {
 	background-color: var( --studio-white );
 }
 
+.mailbox-selection-list__loader-error-container {
+	align-items: center;
+	box-sizing: border-box;
+	display: flex;
+	justify-content: center;
+	margin: 0;
+	min-height: 90vh;
+
+	section {
+		align-items: center;
+		display: flex;
+		flex-direction: column;
+
+		img {
+			width: 153px;
+
+			@include break-medium {
+				width: 194px;
+			}
+		}
+
+		.formatted-header {
+			margin-top: 25px;
+			margin-bottom: 15px;
+
+			@include break-medium {
+				margin-bottom: 32px;
+			}
+
+			.formatted-header__title {
+				color: var( --studio-gray-80 );
```

I think this should be `--color-text`.
```suggestion
				color: var( --color-text );
```
It may not even be necessary, unless the FormattedHeader title specifies a different colour.